src/main/java/core/basesyntax/service/nio/FileService.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service.nio;
+
+import java.util.List;
+import java.util.Map;
+
+public interface FileService {
+    List<String> read(String pathFromRepository);
+
+    void write(String path, Map<String,Integer> map);
```

Comment:
```suggestion
    void write(String path, String content);
```
Create ReportService in main() to make String report using DAO